fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanInlineCOBConfig.java

```diff
@@ -37,13 +37,15 @@
 import org.springframework.batch.core.step.builder.StepBuilder;
 import org.springframework.batch.integration.config.annotation.EnableBatchIntegration;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.transaction.PlatformTransactionManager;
 import org.springframework.transaction.support.TransactionTemplate;
 
 @Configuration
 @EnableBatchIntegration
+@ConditionalOnProperty(value = "fineract.job.loan-cob-enabled", havingValue = "true")
```

Comment:
Should we have `@Conditional(LoanCOBEnabledCondition.class)` instead? I might be wrong but inline COB can be executed only on manager, right?

Reply:
changed